### ISSUE TITLE
Strange dependency on `com.google.vr.cardboard`

### ISSUE THREAD

**Issue by ryan-hodgman** [viromedia/virocore#153]
I'm not sure whether this is intended, but I've observed a crash within the `ViroViewARCore` instantiation code when Google Cardboard is not installed on the device.

When you create an instance using `ViroViewARCore(context, startupListener)` or `ViroViewARCore(context, startupListener, rendererConfig)` the construction code goes straight into an `init()` block, but using any of the Android View constructor methods (such as `ViroViewARCore(context)`) causes the construction code to run the following check...

```Java
if(ContextUtils.getActivity(context) == null) {
    throw new IllegalArgumentException("An Activity Context is required for Viro functionality."); 
} else {
    this.init(context, (ViroViewARCore.StartupListener)null);
}
```

This would be fine, but `ContextUtils` is acquired from `com.google.vr.cardboard.ContextUtils`. Running this code on a device without Google Cardboard installed causes the following exception...

```
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/vr/cardboard/ContextUtils;
                at com.viro.core.ViroViewARCore.<init>(ViroViewARCore.java:374)
                at com.viro.core.ViroViewARCore.<init>(ViroViewARCore.java:362)
                at com.viro.core.ViroViewARCore.<init>(ViroViewARCore.java:352)
                at ...
```

So my questions are: why is this check utilised in some of the construction paths but not others, and why is there a single dependency on Google Cardboard for a utility class? I am anticipating being able to use this "ARCore" view with just ARCore installed, so could we please remove the dependency? :)

**Comment by ryan-hodgman** [comment]
Worth noting for anyone having the same problem that calling one of the ViroCore-specific constructors circumvents the issue (e.g. `ViroViewARCore(context, null, null)`.

**Comment by manbod** [comment]
Whoa, good catch! Legit bug. We've removed that dependency on `ContextUtils`. Should be fixed in an upcoming release that's coming soon. 
In the meantime, what version of ViroCore are you using? Do you have a `gvr_common` directory where you placed the `gvr-common-sdk-xxx.aar` file, along with the `build.gradle` file? If yes, can you ensure it's the right version for the ViroCore version you are using? While we get this bug fix out to release, if you prefer using Android view constructors in your code, download the right version of `gvr_common` sdk from our release page here -> https://virocore.viromedia.com/docs/releases following the instructions for build.gradle. The `ContextUtil` class that you are seeing in the crash comes from this library (which gets bundled into the app), so shouldnt require Cardboard being installed on the device. You can remove this library once you upgrade to our next upcoming release. Apologies for the inconvenience.

**Comment by ryan-hodgman** [comment]
<img width="182" src="https://user-images.githubusercontent.com/26558824/41753874-4b65e424-7613-11e8-8b7c-b4afc56731c7.png">

We're on the latest version (1.7.2). Apologies, I should have specified in my original post.

Great to hear you're working on a fix! Thanks for the tip, but for now I'd prefer calling `ViroViewARCore(context, null, null)` to importing a dependency we don't otherwise use. I'll keep an eye out for the next update. :)

Love your team's work!

**Comment by dam00n** [comment]
This was fixed in ViroCore 1.10.0

Release notes and upgrade instructions here -> https://virocore.viromedia.com/docs/releases
